.github/PULL_REQUEST_TEMPLATE/anvilprod-promotion.md

```diff
@@ -46,7 +46,7 @@ Connected issue: #0000
 - [ ] Pushed PR branch to GitHub
 - [ ] Ran `_select anvilprod.shared && CI_COMMIT_REF_NAME=anvilprod make -C terraform/shared apply_keep_unused` <sub>or this PR is not labeled `deploy:shared`</sub>
 - [ ] Made a backup of the GitLab data volume in `anvilprod` (see [operator manual](../blob/develop/OPERATOR.rst#backup-gitlab-volumes) for details) <sub>or this PR is not labeled `backup:gitlab`</sub>
-- [ ] Ran `_select anvilprod.gitlab && CI_COMMIT_REF_NAME=anvilprod make -C terraform/gitlab apply` <sub>or this PR is not labeled `deploy:gitlab`</sub>
```

Comment:
If the escape hatch was used in the command above, then `anvilprod.gitlab` will not be selected, so generally, selecting it is required. You state

> Remove ineffective command; running it causes errors because the GL instance is down at this point

but i don't understand why the act of selecting the `gitlab` component would be affected by the fact that the instance is down. Please elaborate.

Reply:
Discussed in PL. And agreed the error in the second command is inoffensive.